mail_show_follower/models/mail_mail.py

```diff
@@ -60,96 +60,36 @@ def remove_p(markup_txt):
-                            and (
-                                not x.user_ids
-                                or group_portal in x.user_ids.mapped("groups_id")
-                            )
-                        )
-                    )
-                    if partners_len > 1:
-                        # get partners
-                        cc_internal = True
-                        # else get company in object
-                        if hasattr(obj, "company_id") and obj.company_id:
-                            cc_internal = obj.company_id.show_internal_users_cc
-                        # get company in user
-                        elif mail.env and mail.env.user and mail.env.user.company_id:
-                            cc_internal = (
-                                self.env.user.company_id.show_internal_users_cc
-                            )
-                        if cc_internal:
-                            partners = partners_obj.filtered(
-                                lambda x: x.id not in user_partner_ids
-                                and (
-                                    not x.user_ids
-                                    or any(x.mapped("user_ids.show_in_cc"))
-                                )
-                            )
-                        else:
-                            partners = partners_obj.filtered(
-                                lambda x: x.id not in user_partner_ids
-                                and (
-                                    not x.user_ids
-                                    or group_portal in x.user_ids.mapped("groups_id")
-                                )
-                            )
-                        partners = partners.filtered(
-                            lambda x: not x.user_ids
-                            or x.user_ids  # otherwise, email is not sent
-                            and "email" in x.user_ids.mapped("notification_type")
-                        )
-                        # set proper lang for recipients
-                        langs = list(
-                            filter(
-                                bool,
-                                mail.mapped("recipient_ids.lang")
-                                + [
-                                    mail.author_id.lang,
-                                    self.env.company.partner_id.lang,
-                                ],
-                            )
-                        )
-                        # get show follower text
-                        final_cc = mail.with_context(
-                            lang=langs and langs[0]
-                        )._build_cc_text(partners)
-                        # it is saved in the body_html field so that it does
-                        # not appear in the odoo log
-                        mail.body_html = final_cc + mail.body_html
+        recipients = self.mapped("recipient_ids")
+
+        for mail in self:
```

Comment:
issue: this could be displaying recipients from other mails

```suggestion
        for mail in self:
            recipients = self.recipient_ids
```